konflux-ci/namespace-lister/kustomization.yaml

```diff
@@ -14,6 +14,12 @@ images:
   name: quay.io/konflux-ci/namespace-lister
   newName: quay.io/konflux-ci/namespace-lister
 patches:
+- path: ./patches/with_cachenamespacelabelselector.yaml
+  target:
+    group: apps
+    kind: Deployment
+    name: namespace-lister
+    namespace: namespace-lister
```

Comment:
Can we put it directly in `deployment.yaml` instead?

Reply:
Fixed in be718af68395f0a0802b44791c3a8ce67d03b61b.